src/Tickets/Blocks/Ticket/app/editor/container-header/template.js

```diff
@@ -22,21 +23,20 @@ const TicketContainerHeader = ( {
 		return null;
 	}
 
-	function OnSaleBug( onSale ) {
-		if ( ! onSale ) {
-			return null;
-		}
+	function OnSaleLabel() {
 		return (
 			<div className="tribe-editor__ticket__container-header__sale-label-container">
-				<span className="tribe-editor__ticket__container-header__sale-label">On Sale</span>
+				<span className="tribe-editor__ticket__container-header__sale-label">
+					{ __( 'On Sale', 'event-tickets' ) }
+				</span>
```

Comment:
Nope, let's use the constants as we do here:

Reply:
https://github.com/the-events-calendar/event-tickets/blob/5810f529627495f4701838ec593e6f065f6c905c/src/Tickets/Blocks/Ticket/app/editor/container-content/sale-price/template.js#L174

Reply:
Include a new key for `On Sale` here and use that from the constants: https://github.com/the-events-calendar/event-tickets/blob/6385f9cb5cb70b037e68348d3c15264a3066f69d/src/Tickets/Commerce/Hooks.php#L790

Reply:
This way we only have translatable strings in one place and we are moving towards this pattern for future as well.

Reply:
??

Reply:
If you're referring to the `TICKET_LABELS` constant, that's a different case, as we allow users to use a word other than `Tickets` or `Ticket`. For `On Sale`, I don't feel a constant is necessary.